packages/contracts/contracts/chugsplash/L2ChugSplashDeployer.sol

```diff
@@ -100,19 +101,41 @@ contract L2ChugSplashDeployer is Ownable {
     /***************
      * Constructor *
      ***************/
-    
+
     /**
      * @param _owner Address that will initially own the L2ChugSplashDeployer.
      */
     constructor(
         address _owner
     )
         Ownable()
+        OVM_CrossDomainEnabled(0x4200000000000000000000000000000000000007)
     {
         transferOwnership(_owner);
     }
 
 
+    /*********************
+     * Function Modifier *
+     *********************/
+
+    /**
+     * A modifier like "onlyOwner" but also alternatively allows the owner to call this contract
+     * via an L1 => L2 message.
+     */
+    modifier onlyOwnerOrCrossDomainOwner() {
+        require(
```

Comment:
Is this a gas savings thing to pack so much in a single expression? If so, I understand why there is utility but otherwise readability is more important

Reply:
How would you reformat this to make it more legible?

Reply:
Doesn't the cross domains enabled lib have something that does half of this?

Reply:
Yes, but I don't believe it's possible to compose two function modifiers with an "OR".